docs/DEVGUIDE.md

```diff
@@ -1,10 +1,15 @@
-# An End-to-End Walkthrough
+# Developer's Guide to Service-Catalog
 
-## Prerequisites and Setup
-
-You will need to meet the following prerequisites:
+## Prerequisites
 
+At a minimum you will need:
 * [Docker](https://www.docker.com) installed locally
+* GNU Make
+* [git](https://git-scm.com)
+
+These will allow you to build and test using Docker.
+
+If you want to build without Docker, then you will also need:
```

Comment:
> For example, we're going to have to decide what is the best mechanism to deploy k8s in Docker (e.g. hyperkube?)

I'm not at all suggesting we go down that rabbit hole right now.

I only felt that because a very general list of pre-reqs got broken into two, with each explicitly labeled as the in-container vs out-of-container build reqs, that matters like "you need a working k8s cluster," don't belong on either of those lists.

Whether this is tackled in this PR or in a follow-up is up to you. I won't let this block my LGTM.